openapi/api.yaml

```diff
@@ -146,6 +146,67 @@ paths:
                   $ref: '#/components/schemas/SpecMentor'
         '400':
           description: The ?after query parameter isn't a valid date in ISO YYYY-MM-DD format.
+  /features/{feature_id}/{stage_id}/intent:
+    get:
+      summary: Get the HTML body of an intent draft
+      operationId: getIntentBody
+      parameters:
+        - name: feature_id
+          in: path
+          description: Feature ID
+          required: true
+          schema:
+            type: integer
+        - name: stage_id
+          in: path
+          description: Stage ID
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Intent draft body.
+          content:
+            text/plain:
```

Comment:
Instead of `text/plain`, let's do something more like `MessageResponse` where there is a single string field, but we can more easily add other fields later.

Reply:
Added a new `GetIntentResponse` definition with the `subject` and `email_body` fields.